tests/unit/kernels_reference.cpp

```diff
@@ -104,6 +104,27 @@ void dw::reference::CPUReLUBackward(const float* in, const float* grad_output,
                    [](float in, float go) { return in > 0.0 ? go : 0.0;});
 }
 
+void dw::reference::CPUELUForward(const Tensor& in, Tensor& out, float alpha) {
+    float* raw_in = in.data();
+    float* raw_out = out.data();
+
+    for (size_t i = 0; i < in.total(); i++) {
+        raw_out[i] = raw_in[i] < 0.f ? alpha * (std::exp(raw_in[i]) - 1) : raw_in[i];
```

Comment:
Just wondering why it isn't `transofrm` as for `CPUELUBackward` let's keep consistence. Chose the single solution for that

Reply:
Done.